src/utils.rs

```diff
@@ -27,9 +27,9 @@ mod native {
 
     use crate::Result;
 
-    fn read_urdf(path: &str) -> Result<(urdf_rs::Robot, String)> {
+    fn read_urdf(path: &str, args: &[(String, String)]) -> Result<(urdf_rs::Robot, String)> {
```

Comment:
urdf does not accept args. args is only for xacro.
So how about rename to xacro_args?

```suggestion
    fn read_urdf(path: &str, xacro_args: &[(String, String)]) -> Result<(urdf_rs::Robot, String)> {
```